cornflow/endpoints/user.py

```diff
@@ -84,7 +87,7 @@ def delete(self, user_id):
         user_obj = UserModel.get_one_user(user_id)
         if user_obj is None:
             raise ObjectDoesNotExist()
-        if user_obj.super_admin and not self.is_super_admin():
+        if user_obj.is_service_user():
             raise NoPermission()
```

Comment:
service user? you have `ROLES_WITH_ACCESS = [ADMIN_ROLE]` I guess you can delete this part if we already enforce roles somewhere else.

Reply:
Its right. Will do.